apps/etterlatte-saksbehandling-ui/client/src/components/behandling/beregningsgrunnlag/BeregningsgrunnlagBarnepensjon.tsx

```diff
@@ -146,14 +164,24 @@ const BeregningsgrunnlagBarnepensjon = () => {
               success: (trygdetider) => (
                 <>
                   {trygdetider.length > 1 && (
-                    <BeregningsgrunnlagFlereAvdoede redigerbar={redigerbar} trygdetider={trygdetider} />
+                    <BeregningsgrunnlagFlereAvdoede
+                      redigerbar={redigerbar}
+                      trygdetider={trygdetider}
+                      tidligsteAvdoede={tidligsteAvdoede}
+                    />
                   )}
-                  {trygdetider.length <= 1 && (
+                  {trygdetider.length == 1 && (
```

Comment:
Riktig å kreve at length == 1? Man må vel ha trygdetid for å kunne registrere beregningsmetode?

Reply:
Fikser 3 = :)